dashboard/client/pages/login.tsx

```diff
@@ -1,14 +1,44 @@
 import * as React from 'react'
 
-import GoogleButton from '../components/GoogleButton'
+import Router from 'next/router'
+import { auth } from '../lib/gql/query'
+
 import Layout from '../components/Layout'
 
-const Login: React.SFC = () => (
-  <Layout>
-    <main className="main" id="main__login">
-      <GoogleButton />
-    </main>
-  </Layout>
-)
+const Login = ({ query }) => {
+  const [user, setUser] = React.useState()
+
+  React.useEffect(() => {
+    const isBrowser = typeof window !== undefined
```

Comment:
should be `'undefined'`

Reply:
what do u mean??

Reply:
`typeof` operator returns a string of the operand's type. in this case, it's `'undefined'`

Reply:
ooo thank you!